src/Tribe/Aggregator/Tabs/Abstract.php

```diff
@@ -97,6 +97,7 @@ public function handle_submit() {
 		$record = Tribe__Events__Aggregator__Records::instance()->get_by_origin( $post_data['origin'] );
 
 		$meta = [
+			'import_name'   => empty ( $post_data['import_name'] ) ? '' : sanitize_text_field( trim( $post_data['import_name'] ) ),
```

Comment:
:no_entry_sign: Space before opening parenthesis of function call prohibited

<sub>hash: 86aefc357916baaa40d10d226eb38706</sub>